codersdk/certificateUtils.go

```diff
@@ -0,0 +1,53 @@
+package codersdk
```

Comment:
I think we should avoid moving this logic into `codersdk` since it's pretty much the only public API we have, and they're not useful outside the server (i.e. `coderd`).

Reply:
Ah

Didn't notice `cli` imports from `coderd`

Moved `certificateUtils.go` over to `coderd`

Sorry still learning the structure 😄

Reply:
Ah never mind I'm still at a dependency cycle...

Is there a package where these utils can go so they don't cause a dependency cycle? (Since this file is depended on by both `cli` _and_ `coderd`)